.github/workflows/main.yml

```diff
@@ -29,6 +29,62 @@ jobs:
           python-version: "3.11"
           install-just: true
       - name: Run tests
-#       env:  # Add environment variables required for tests
+        run: just test
+
+  test-package-build:
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
We only need to build a package for job server to consume, so only 3.11

The python 3.8 code that is include is not imported or executed 3.11. The tests execute it using the python docker image, which is 3.8

Reply:
ooh perfect, fixed!